packages/storage/src/implementation/backoff.ts

```diff
@@ -58,22 +60,31 @@ export function start(
   }
 
   function callWithDelay(millis: number): void {
-    timeoutId = setTimeout(() => {
-      timeoutId = null;
+    retryTimeoutId = setTimeout(() => {
+      retryTimeoutId = null;
       f(handler, canceled());
     }, millis);
   }
 
+  function clearGlobalTimeout() {
+    if(globalTimeoutId){
```

Comment:
Please fix.

This code also needs formatting.

Reply:
Done.